File: PyCharm/circulo.py
import math
from figurasGeometricas import FigurasGeometricas


class Circulo(FigurasGeometricas):
    def __init__(self):
        self.__radio = 0

    def setRadio(self,valor):
        self.__radio = valor

    def perimetro(self):
        return math.pi * self.__radio * 2

    def area(self):
        return math.pi * math.pow(self.__radio, 2)

File: NetBeansProjects/Calderon_Raymond/src/Circulo/Circulo.java

package Circulo;

import java.text.DecimalFormat;
import java.util.Scanner;

/**
 *
 * @author Raymond
 */
public class Circulo {

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
            Double radio;
            
            Scanner t = new Scanner(System.in);
            System.out.println( "Ingrese el Radio : ");
            radio = t.nextDouble();
        
        Double Area = Math.PI * radio * radio;
        Double Circunferencia = Math.PI * (radio + radio);
        
            DecimalFormat df = new DecimalFormat ("#,##");
            System.out.println("El area del circulo es: " + df.format(Area) );
            System.out.println("La circunferencia del circulo es: "+ df.format(Circunferencia) );
        
    }
    
}
